File: nashvilletheatre.ui/src/components/pages/Cart/LineItem.js
import React from 'react'

class Cart extends React.Component {

  render() {
    const { item } = this.props
    return (
      <tr>
          <td>{item.itemName}</td>
          <td>{item.showDateTime}</td>
          <td>{item.quantity}</td>
          <td>{item.itemPrice}</td>
      </tr>
    );
  }
}
export default Cart;

File: NashvilleTheatre/Models/LineItem.cs
using System;
namespace NashvilleTheatre.Models
{
    public class LineItem
    {
        public int CartId { get; set; }
        public int LineItemId { get; set; }
        public string LineItemType { get; set; }
        public int ProductId { get; set; }
        public int Quantity { get; set; }
        public DateTime DateAdded { get; set; }
    }

    public class ShowLineItem
    {
        public int LineItemId { get; set; }
        public string ItemName { get; set; }
        public DateTime ShowDateTime { get; set; }
        public int Quantity { get; set; }
        public decimal ItemPrice { get; set; }
    }

    public class SubscriptionLineItem
    {
        public int LineItemId { get; set; }
        public string ItemName { get; set; }
        public decimal ItemPrice { get; set; }
        public int Quantity { get; set; }
    }

}
